<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainUi</class>
 <widget class="QWidget" name="MainUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainUi</string>
  </property>
  <widget class="QPushButton" name="startWebCatchPB">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>抓取</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="webUrlLE">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>591</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="logLE">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>750</y>
     <width>1150</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Url:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="resultTE">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>741</width>
     <height>511</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="currentUrlL">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>741</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="filterResultTE">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>220</y>
     <width>351</width>
     <height>511</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>941</width>
     <height>130</height>
    </rect>
   </property>
   <property name="title">
    <string>过滤器</string>
   </property>
   <widget class="QLineEdit" name="filterLE">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="useRegExpFilterRB">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>正则筛选</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QLineEdit" name="prefixFilterLE">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="suffixFilterLE">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>30</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>前缀：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>54</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>后缀：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>54</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>内容：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="refiltPB">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>30</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>重新筛选</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="useImportFilterRB">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>文件内容</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QLineEdit" name="importFilterLE">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="importFilterPB">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>80</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>筛选文件内容</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="usedFindFilterRB">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>文本内容</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>190</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>筛选结果</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveWebPB">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>保存原网页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveFiltResultPB">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>90</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>保存筛选结果</string>
   </property>
  </widget>
  <widget class="QPushButton" name="importWebPB">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>导入代替抓取</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
